huxley/www/js/components/Shaker.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2011-2017 Berkeley Model United Nations. All rights reserved.
+ * Use of this source code is governed by a BSD License (see LICENSE).
+ */
+
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var cx = require('classnames');
+
+require('css/Shaker.less');
+
+var Shaker = React.createClass({
+  childContextTypes: {
```

Comment:
Does this need to be wrapped into a component? A function stored in the utils folder might be simpler if it were possible, but I don't know how that would work with the CSS.